BookingsApi/BookingsApi.IntegrationTests/Api/V2/Hearings/UpdateHearingsInGroupV2Tests.cs

```diff
@@ -114,8 +114,10 @@ public async Task should_update_hearings_in_group()
                 AssertParticipantsUpdated(updatedHearing, requestHearing);
                 AssertEndpointsUpdated(updatedHearing, requestHearing);
                 AssertJudiciaryParticipantsUpdated(updatedHearing, requestHearing);
-                AssertEventsPublished(updatedHearing, requestHearing, existingParticipantsUpdated: 1);
+                AssertEventsPublished(updatedHearing, requestHearing, 1);
             }
+            var requestFirstHearing = request.Hearings.First(x => x.HearingId == updatedHearings[0].Id);
+            AssertNotificationEvents(updatedHearings[0], requestFirstHearing, existingParticipantsUpdated: 1);
```

Comment:
We need to add this assertion to the V1 tests too

Reply:
done